webpack/scenes/ContentViews/Details/Filters/Add/CVFilterAddModal.js

```diff
@@ -0,0 +1,160 @@
+import React, { useState } from 'react';
+import useDeepCompareEffect from 'use-deep-compare-effect';
+import { Redirect } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { STATUS } from 'foremanReact/constants';
+import PropTypes from 'prop-types';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { Modal, ModalVariant, Form, FormGroup, TextInput, ActionGroup, Button, Radio, TextArea,
+  Split, SplitItem, Select, SelectOption } from '@patternfly/react-core';
+import { createContentViewFilter } from '../../ContentViewDetailActions';
+import {
+  selectCreateContentViewFilter, selectCreateContentViewFilterStatus,
+  selectCreateContentViewFilterError,
+} from '../../../Details/ContentViewDetailSelectors';
+import { FILTER_TYPES } from '../../../ContentViewsConstants';
+import ContentType from '../ContentType';
+
+const CVFilterAddModal = ({ cvId, show, setIsOpen }) => {
+  const selectOptions = Object.keys(FILTER_TYPES).map(item =>
+    <SelectOption key={item} value={item}><ContentType type={item} /></SelectOption>);
```

Comment:
Prefer to have this JSX in the return, not up here with the data values.  I don't see a need to assign it to a variable.